src/app/onion/learning-object-builder/builder-store.service.ts

```diff
@@ -193,6 +193,25 @@ export class BuilderStore {
     this.learningObjectEvent.next(this.learningObject);
   }
 
+  /**
+   * Retrieves the learning objects children
+   *
+   */
+  async getChildren() {
+    return await this.learningObjectService.getChildren(
+      this.learningObject.id
+    );
+    
+  }
+
+  /**
+   * Sets the learning objects children after they have been reorderd
+   */
+  async setChildren(children: string[]) {
+    console.log(this.learningObject.name);
```

Comment:
We shouldn't be checking console logs into version control.

```suggestion
    console.log(this.learningObject.name);
```